cost/aws/elb/clb_unused/AWS_Delete_Unused_CLB.pt

```diff
@@ -134,7 +134,7 @@ script "js_aws_clb_filter_map", type: "javascript" do
     var tagKeyValue=""
     for(var j=0; j < tags.length; j++){
       tag = tags[j]
-      if(param_exclude_tags.indexOf(tag['tagKey']+'='+tag['tagValue']) !== -1){
+      if(((param_exclude_tags.join()+',').toLowerCase()).indexOf((tag['tagKey']+'='+tag['tagValue']+',').toLowerCase()) !== -1){
```

Comment:
Create a temporary array for the param_exclude_tags list with lowercase keys and use array indexOf. Doing string search with a comma`,` delimiter causes weird corner cases.

Reply:
Thanks for suggestion, we have updated accordingly ..

Reply:
@avinashbhashyam-rs  can you resolve this review if it meets your suggestion